helix-term/src/commands.rs

```diff
@@ -6806,6 +6815,10 @@ fn jump_to_label(cx: &mut Context, labels: Vec<Range>, behaviour: Movement) {
                 } else {
                     range.with_direction(Direction::Forward)
                 };
+                {
+                    let (view, doc) = current!(cx.editor);
+                    push_jump(view, doc);
+                }
```

Comment:
I'm not sure if this should be done for `gw`. `gw` doesn't necessarily move you very far in the file, like `gc` or `gb`, so a jump seems less useful